### DISCUSSION TITLE
Should we rename `theme` prop of the Halstack Context?

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#1567] [Ideas] [answered]
Reading the [Halstack Provider page](https://developer.dxc.com/halstack/next/utilities/halstack-provider/), I wondered: Why do we call the opinionated theme prop `theme`? On every page of the documentation we refer to it as an opinionated theme, that's a constant adopted in our Halstack vocabulary. Shouldn't we rename it as `opinionantedTheme` for a consistent API?

**Comment by kukoc** [comment]
I agree. Especially when we have another prop called advanced theme. The Api would be more meaningful if we'd use opininatedTheme. However this is a breaking change, as any change in our components api. We need to plan this change so we still provide a smooth deve experience.

**Reply by GomezIvann** [reply]
Agree. It's a huge change that we need to deliver carefully. I still believe it's an appropriate change.

**Reply by kukoc** [reply, marked as answer]
Let's plan this breaking change and do our best to get everybody aware of this change.
